sensorsafrica/router.py

```diff
@@ -3,7 +3,7 @@
 from django.conf import settings
 
 class ReplicaRouter:
-    read_replicas = settings.READ_DATABASE_URLS
+    read_replicas = list(settings.DATABASES.keys())
```

Comment:
```suggestion
    databases = list(settings.DATABASES.keys())
```

I take it this includes **_everything_** i.e. `default` + all `read_replica_xxx`.

Reply:
Yes this includes all the databases @kilemensi

Reply:
Shouldn't call it `read_replicas` then @thepsalmist